package.json

```diff
@@ -16,10 +16,10 @@
     "@oclif/plugin-command-snapshot": "^3.1.3",
     "@oclif/plugin-commands": "^2.1.0",
     "@oclif/plugin-help": "^5.1.12",
-    "@salesforce/cli-plugins-testkit": "^2.3.0",
-    "@salesforce/dev-config": "^3.0.1",
-    "@salesforce/dev-scripts": "^2.0.0",
-    "@salesforce/plugin-command-reference": "^1.3.20",
+    "@salesforce/cli-plugins-testkit": "^2.3.11",
+    "@salesforce/dev-config": "^3.1.0",
+    "@salesforce/dev-scripts": "^3.1.0",
```

Comment:
This was a major version bump but I couldn't find any release notes saying what the breaking change was (if any).